examples/server_profile_template_without_i3s.pp

```diff
@@ -0,0 +1,207 @@
+################################################################################
```

Comment:
Regarding the naming of the file, How is this different from the existing example for SPT? In my opinion, we should include this create block in the existing SPT example, so that this extra file can be avoided.